### PR TITLE
Fix for Rails 7.2

### PR DESCRIPTION
**Pull request by jbennett**
Rails 7.2 adds a required `connection` param. This passes whatever comes in to the super method.

### CODE REVIEW

**Comment by subzero10** [comment]
Hey @jbennett, seems like tests are failing on Rails 7.2. Would you like to take a look?

File: app/models/heya/campaign_membership.rb [hunk 1]
```diff
@@ -52,9 +52,9 @@ class CampaignMembership < ApplicationRecord
     scope :to_process, ->(now: Time.now, user: nil) {
       upcoming
         .where(<<~SQL, now: now.utc, user_type: user&.class&.base_class&.name, user_id: user&.id)
-          ("heya_campaign_memberships".last_sent_at <= (TIMESTAMP :now - make_interval(secs := "heya_steps".wait)))
+          ("heya_campaign_memberships".last_sent_at <= (:now::timestamp - make_interval(secs := "heya_steps".wait)))
           AND (
-            (:user_type IS NULL OR :user_id IS NULL)
+            (:user_type::text IS NULL OR :user_id::int IS NULL)
```
Comment (olivier-thatch) [inline app/models/heya/campaign_membership.rb:57]: Can this change be omitted? We use strings for user IDs and this cast breaks Heya for us.

Comment (jbennett) [reply app/models/heya/campaign_membership.rb]: I've changed the query to remove the need for type casting. Let me know if this works for you now

Comment (olivier-thatch) [reply app/models/heya/campaign_membership.rb]: 👍  It works, thanks for addressing this quickly!

**Review by joshuap — APPROVED** [review]
This looks good to me, thanks! I'll try to get it merged/released this weekend.

**Pull request merged**